### PR TITLE
[TASK] Hide package label in page layout selector if no page layouts are provided

### PR DESCRIPTION
**Pull request by georg-tiefenbrunn**
This patch makes ``PageLayoutSelector->renderOptions()`` return an empty string and thus does not render the package label if the provider extension does not provide any page layouts. See https://github.com/FluidTYPO3/flux/issues/839#issuecomment-105763399

### CODE REVIEW

File: Classes/Backend/PageLayoutSelector.php [hunk 1]
```diff
@@ -133,19 +133,21 @@ protected function renderInheritanceField(array $parameters) {
...
 		$selector = '';
-		$extensionKey = ExtensionNamingUtility::getExtensionKey($extension);
-		if (FALSE === ExtensionManagementUtility::isLoaded($extensionKey)) {
-			$groupTitle = ucfirst($extension);
-		} else {
-			$emConfigFile = ExtensionManagementUtility::extPath($extensionKey, 'ext_emconf.php');
-			require $emConfigFile;
-			$groupTitle = $EM_CONF['']['title'];
-		}
+		if (!empty($group)) {
```
Comment (NamelessCoder) [inline Classes/Backend/PageLayoutSelector.php]: `empty($group) === FALSE` please ;)

Comment (georg-tiefenbrunn) [inline Classes/Backend/PageLayoutSelector.php]: Ooops! CGL :) But shouldn't that be ``FALSE === empty($group)``?

Comment (NamelessCoder) [inline Classes/Backend/PageLayoutSelector.php]: Of course, correct! I'm doing code reviews in four different CGLs this week, sorry about the confusion.

**Comment by georg-tiefenbrunn** [comment]
Sorry. Using ``git commit --amend --no-edit && git push --force`` from the [docs]( https://github.com/FluidTYPO3/documentation/blob/0a8bf94ae4c3951206e1beca0e8e805dcbaab3e5/6.DeveloperResources/6.2.Contributing.md#how-fix-your-pull-request) totally broke this PR. I'll start from scratch ...

**Pull request closed without merging**